lib/phlex/rails/streaming.rb

```diff
@@ -17,10 +17,29 @@ def stream(view)
 		self.response_body = Enumerator.new do |buffer|
 			view.call(buffer, view_context: view_context)
 		rescue => e
-			buffer << %('">)
-			buffer << view_context.javascript_tag(nonce: true) { %(window.location = "/500.html").html_safe }
+			raise (e) if Rails.env.test?
 
-			raise e
+			debug_middleware = ActionDispatch::DebugExceptions.new(
+				proc { |env| raise(e) },
+				response_format: :html
+			)
+
+			_debug_status, _debug_headers, debug_body = debug_middleware.call(request.env)
```

Comment:
We can call the `ActionDispatch::DebugExceptions` middleware to get the debug page HTML, which we can send via JavaScript to the streaming client.